src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -47,7 +47,7 @@ public class ConsoleRunner
         public static readonly int INVALID_ASSEMBLY = -2;
         //public static readonly int FIXTURE_NOT_FOUND = -3;    //No longer in use
         public static readonly int INVALID_TEST_FIXTURE = -4;
-        public static readonly int UNLOAD_ERROR = -5;
+        //public static readonly int UNLOAD_ERROR = -5;         //No longer in use
```

Comment:
LGTM, but we should just remove it if we are removing it.

Reply:
I wanted to leave it there to make sure we don't end up reusing -5 for something else in a couple of releases time. I can remove if  you disagree - what do you think?

Reply:
Good enough reason 😄